frontend/src/router/index.ts

```diff
@@ -57,7 +57,7 @@ const router = createRouter({
 			component: () => import('../views/NewChannelView.vue'),
 		},
 		{
-			path: '/profile',
+			path: '/profile/:id',
```

Comment:
Would we need both `/profile/` and `/profile/:id`
one for your own profile and one for others their profile?

Or will `/settings` (/me or whatever it will be) be "editing your own profile" and `/profile/:id` with your own ID be the page where you can view your own profile?

Reply:
We can maybe add an option to "setting for viewing your own profile? Don't mind if it's with `/:id` or without, I think we can just reuse the `/:id` path? Not sure that it makes sense to a have completely diff.